libs/build_cmakemodules.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -eux
+
+name="cmakemodules"
+repo=${1:-${STACK_cmakemodules_repo:-"NOAA-EMC"}}
```

Comment:
NOAA-EMC is the only repo from which we will install in the stack.
Please remove $repo from the `prefix` path and modulefiles.

Reply:
Done in 81e539f4a6443c4253b9e4825eeeebcd28bda884